src/dis.c

```diff
@@ -137,7 +137,8 @@ static const char *decode_jump8(const uint8_t *ip, const char *ins, uint16_t reg
 static const char *decode_far(const uint8_t *ip, const char *ins)
 {
     fillbytes(ip, 5);
-    sprintf(IPOS, "%-7s %04X:%04X", ins, ip[4] * 256 + ip[3], ip[2] * 256 + ip[1]);
+    sprintf(IPOS, "%-7s %04X:%04X", ins,
+            (unsigned)(ip[4] * 256 + ip[3]), (unsigned)(ip[2] * 256 + ip[1]));
```

Comment:
IMHO, this should be `ip[4] * 256U + ip[3], ip[2] * 256H + ip[1]` , it is cleaner.